PublishWorkflow/PublishWorkflowChecker.php

```diff
@@ -132,36 +132,4 @@ public function isGranted($attributes, $object = null)
 
         return $this->accessDecisionManager->decide($token, $attributes, $object);
     }
-
-    private function getTokenStorage()
```

Comment:
looks like we forgot to delete these in master - we do not use them anymore as we inject the token storage and auth checker explicitly.